backend/api/views.py

```diff
@@ -76,7 +76,7 @@ def post(self, request):
             return Response(
                 {
                     "valid": True,
-                    "response": json.loads(serializer.data.get("auditee_uei")),
+                    "response": json.dumps(serializer.data.get("auditee_uei")),
```

Comment:
Should we update (or create) tests around this to assert that the expected fields are coming back?